src/python/pants/backend/shell/target_types.py

```diff
@@ -431,10 +431,14 @@ class ShellCommandLogOutputField(BoolField):
 class ShellCommandWorkdirField(StringField):
     alias = "workdir"
     default = None
-    help = (
+    help = softwrap(
         "Sets the current working directory of the command, relative to the project root. If not "
-        "set, use the project root. To use the location of the `BUILD` file, use "
-        '`f"{build_file_dir()}"`.'
+        "set, use the project root.\n\n"
+        "The following special values are supported:\n\n"
+        "* `.`: the location of the `BUILD` file that defines this target.\n"
+        "* `/`: the project root.\n"
+        "\n"
+        "Directories may be specified relative to either of these special values."
```

Comment:
I'm not sure if the relative terminology here is helpful or confusing together with the `/` as a "special value".

How about describing it as the path is rooted at the project root and that relative paths are relative to the location of the BULD file, and that paths are implicitly rooted unless starting with a `.`?

Reply:
sure.

Reply:
See https://github.com/pantsbuild/pants/pull/17928/commits/86bcd6c80ac31a0416e1b08395365916e7408482